app/src/componentLibrary/button/button.scss

```diff
@@ -86,6 +86,17 @@
   background: none;
   color: $COLOR--topaz-2;
   line-height: 22px;
+
+  svg * {
+    fill: $COLOR--topaz-2;
```

Comment:
Can you explain why you touched this file? Looks like you don't use Button component

Reply:
There were ESlint problems in that file. ESlint rules says : that subsequence of selectors must be the following 
focus=>active=>hover.